internal/auth/provider/github/github.go

```diff
@@ -0,0 +1,80 @@
+package github
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/google/go-github/v51/github"
+	"github.com/google/uuid"
+	"github.com/rapidmidiex/rmx/internal/auth"
+	"golang.org/x/oauth2"
+
+	ghOAuth "golang.org/x/oauth2/github"
+
+	"github.com/zitadel/oidc/v2/pkg/client/rp"
+	httphelper "github.com/zitadel/oidc/v2/pkg/http"
+	"github.com/zitadel/oidc/v2/pkg/oidc"
+)
+
+var (
+	key = []byte("test1234test1234")
```

Comment:
We should grab these cookie keys from the env/cfg.
```suggestion
	key = []byte(cfg.google.cookieKey)
```

Reply:
yup definitely.